number_of_robots: 1

quadrotor_name: dragonfly11
mass: 0.250 # redundant of robot_mass
robot_mass: 0.250
payload_mass: 0.020
cable_length: 0.5
my_index: 0

k_pos_0: {x: 1.8, y: 1.8, z: 6.0}
ki_pos_0: {x: 0.0, y: 0.0, z: 0.0}

max_pos_0_int = 0.5

k_vel_0: 0.7
k_R_0: 0.0
k_Omega_0: 0.0
k_q: 50.0
k_w: 0.0

pl_vel_lowpass_alpha: 0.05
pl_vel_jumpmax: 0.5

attachment:
  rho_0: {x: 0.000, y: 0.000, z: 0.000}
  #rho_1: {x: -0.500, y: 0.866, z: 0.000}
  #rho_2: {x: -0.500, y: -0.866, z: 0.000}
